### PR TITLE
Synopsysctl version

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/489#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/constants.go [hunk 1]
```diff
@@ -21,9 +21,6 @@ under the License.
 
 package synopsysctl
 
-// SynopsysctlVersion is the current version of the synopsysctl utility
-const SynopsysctlVersion string = "2019.6.0"
-
 // DefaultOperatorImage is Synopsys Operator image that is deployed by default
 const DefaultOperatorImage string = "docker.io/blackducksoftware/synopsys-operator:2019.6.0"
```
Comment (rrati) [inline pkg/synopsysctl/constants.go]: Can we use the version as the image tag here?

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/489#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/489#pullrequestreview-252261396" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/489#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/489#pullrequestreview-252339272" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/489#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/489#issuecomment-504058570" title="LGTM">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@rrati: #489 failed to apply on top of branch "master":
```Applying: synopsysctl and operator ui version * synopsysctl version * set version in Operator UI * beautify CSS and fixed the footer * including aliases for all commands and also fixed the default operator image
Using index info to reconstruct a base tree...
M	cmd/operator-ui/go.mod
M	pkg/synopsysctl/cmd_create.go
M	pkg/synopsysctl/cmd_delete.go
M	pkg/synopsysctl/cmd_deploy.go
M	pkg/synopsysctl/cmd_describe.go
M	pkg/synopsysctl/cmd_edit.go
M	pkg/synopsysctl/cmd_get.go
M	pkg/synopsysctl/cmd_root.go
M	pkg/synopsysctl/cmd_start.go
M	pkg/synopsysctl/cmd_stop.go
M	pkg/synopsysctl/cmd_update.go
M	pkg/synopsysctl/constants.go
Falling back to patching base and 3-way merge...
Auto-merging pkg/synopsysctl/constants.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/constants.go
Auto-merging pkg/synopsysctl/cmd_update.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_update.go
Auto-merging pkg/synopsysctl/cmd_stop.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_stop.go
Auto-merging pkg/synopsysctl/cmd_start.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_start.go
Auto-merging pkg/synopsysctl/cmd_root.go
Auto-merging pkg/synopsysctl/cmd_get.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_get.go
Auto-merging pkg/synopsysctl/cmd_edit.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_edit.go
Auto-merging pkg/synopsysctl/cmd_describe.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_describe.go
Auto-merging pkg/synopsysctl/cmd_deploy.go
Auto-merging pkg/synopsysctl/cmd_delete.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_delete.go
Auto-merging pkg/synopsysctl/cmd_create.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_create.go
Auto-merging cmd/operator-ui/go.mod
CONFLICT (content): Merge conflict in cmd/operator-ui/go.mod
error: Failed to merge in the changes.
Patch failed at 0001 synopsysctl and operator ui version * synopsysctl version * set version in Operator UI * beautify CSS and fixed the footer * including aliases for all commands and also fixed the default operator image

```

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/489#issuecomment-504168678):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**